modules/installation-azure-user-infra-completing.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-user-infra.adoc
+
+[id="installation-azure-user-infra-completing_{context}"]
+= Completing an Azure installation on user-provisioned infrastructure
+
+After you start the {product-title} installation on Microsoft Azure
+user-provisioned infrastructure, you can monitor the cluster events until the
+cluster is ready.
+
+.Prerequisites
+
+* Deploy the bootstrap machine for an {product-title} cluster on user-provisioned Azure infrastructure.
+* Install the `oc` CLI and log in.
+
+.Procedure
+
+* Complete the cluster installation:
++
+----
+$ ./openshift-install --dir=<installation_directory> wait-for install-complete <1>
```

Comment:
The UPI github doc does not use the --dir=<installation_directory> option, so at least one of the documents would need to be adopted to follow that convention if that's how we want to set up the product docs.

Reply:
This is a pattern used throughout our official installer docs, so I'd elect to update the Github docs if we want them to be uniform.

Reply:
@fabianofranz Can we support the `./openshift-install --dir=<installation_directory>` pattern with your docs and templates as well?

Reply:
@codyhoag @e-tienne on it.

Reply:
I've tested the product docs using `--dir=<installation_directory>` with no major issue so far.